qa/rpc-tests/pruning.py

```diff
@@ -314,7 +314,7 @@ def wallet_test(self):
         print ("Syncing node 5 to test wallet")
         connect_nodes(self.nodes[0], 5)
         nds = [self.nodes[0], self.nodes[5]]
-        sync_blocks(nds)
+        sync_blocks(nds, timeout=300)
```

Comment:
Forgot about that, thanks -- bumped the wait to 5 seconds.  No objection to teaching sync_blocks to be smarter as well, but I'd like this merged asap to get the test suite working again.